terraform/modules/mimir/main.tf

```diff
@@ -1,39 +1,36 @@
 # TODO: Replace s3_integrator resource to use its remote terraform module once available
 resource "juju_application" "s3_integrator" {
-  name  = var.s3_integrator_name
+  name  = "s3-integrator"
   model = var.model_name
-  trust = true
 
   charm {
     name    = "s3-integrator"
-    channel = var.channel
+    channel = "2/edge" # hardcoding instead of using var.channel
```

Comment:
Update comment re new approach with secret? Is there a minimum rev or `LIBPATCH.LIBAPI` we can specify here?

Reply:
I don't think there's a minimum revision to specify, `2/edge` is channel we want here

`LIBPATCH.LIBAPI` is a charm library concept, I'm not sure how it's related